packages/neos-ui/src/Containers/Drawer/style.css

```diff
@@ -28,6 +28,12 @@
     text-align: left;
     font-size: 15px;
 }
+/* Set an explicit width for the icons so that the chars are starting */
+/* on the same place for every MenuItem */
+.drawer__menuItemGroupBtn i,
+.drawer__menuItemBtn i {
+    width: 2em;
```

Comment:
@mstruebing just wondering, why do we sometimes set the widths in pixels, and sometimes in em?

Reply:
I have set them to em because I thought it was wiser if we change the font-size somewhere

Reply:
I see, thanks for explaining